cmd/Kconfig

```diff
@@ -2183,6 +2183,16 @@ config CMD_FIOVB
 	    fiovb write_rb - write rollback index
 	    fiovb write_pvalue - write to RPMB
 	    fiovb read_pvalue - read from RPMB
+
+config CMD_SCP03
```

Comment:
I guess the first thing that you'll be asked while upstreaming that (if it's planned) - to cover it with python tests :)

Reply:
well, we will have to cross that bridge when they ask :)